### PR TITLE
Displaying trainers

### PR DESCRIPTION
**Pull request by matt712**

### CODE REVIEW

**Review by matt712 — COMMENTED** [review]
ignore this

File: src/AccountSettings.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    constructor(props){
+        super(props);
+        this.state = {message:""};
+    }
+    setPassword =(e)=>{
+        e.preventDefault();
+        let pword1 = this.state.password;
+        let pword2 = this.state.password2;
+        if(pword1===pword2){
+            this.setState({password:pword1, password2: pword2, message: "Password changed successfully"});
```
Comment (JHarry444) [inline src/AccountSettings.js]: Doesn't check for existing password.

Comment (JHarry444) [inline src/AccountSettings.js]: Unnecessarily setting state for passwords.

File: src/AccountSettings.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        const value = e.target.value;
+        const name = e.target.name;
+        this.setState({[name]: value, message: ""});
+    }
+    render(){
+        return(
+            <div className="passwordSet">
+                <form onSubmit={this.setPassword}>
+                    New password:
+                    <input type="password" name="password" placeholder="a password" id="pwordbox1-change" onChange={this.updateState}/><br/>
```
Comment (JHarry444) [inline src/AccountSettings.js]: Change placeholder

File: src/AddTrainer.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+class AddTrainer extends Component{
+    constructor(props){
+        super(props);
+        this.state = {message:"", formDisplay:true};
+    }
+    handleSubmit =(e)=>{
+        e.preventDefault();
+        let body = { email:this.state.email, password:this.state.password, confirmPassword: this.state.password2, trainerName: this.state.name };
+        let url = "http://localhost:8080/createAccount";
```
Comment (JHarry444) [inline src/AddTrainer.js]: Change hard-coded strings to use constants.

File: src/AddTrainer.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        this.setState({message:"", formDisplay:true})
+    }
+    render(){
+        if(this.state.formDisplay===true){
+        return(
+            <div className="addTrainer">
+            <br/>
+            <form onSubmit={this.handleSubmit}>
+                <p className="form-font">Trainer name: </p>
+                <input type="text" name="name" placeholder="Bob Bobson" id="namebox-addtrainer" onChange={this.updateState}/><br/>
```
Comment (JHarry444) [inline src/AddTrainer.js]: Fix placeholders.

File: src/App.js [hunk 5]
```diff
@@ -1,28 +1,61 @@
...
+import TrainerPage from './TrainerPage';
+import AdminPage from './AdminPage';
+import Axios from 'axios';
 
 class App extends Component {
+  constructor(props){
+    super(props);
+    this.state = {route:"login", user:""};
+  }
+  handleLogin = (username, password1) =>{
```
Comment (JHarry444) [inline src/App.js:14]: No.

File: src/DisplayTrainers.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import './App.css';
+import Axios from 'axios';
+
+class DisplayTrainers extends Component{
+    constructor(props){
+        super(props);
+        var url="http://localhost:8080/getAllAccounts"
```
Comment (JHarry444) [inline src/DisplayTrainers.js]: Change to constant.

**Review by JHarry444 — CHANGES_REQUESTED** [review]
Fix security, hard-coded string, etc.

**Review by JHarry444 — APPROVED** [review]

**Pull request merged**